src/main/webapp/css/user-page.css

```diff
@@ -13,6 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+body {
+  background-color: #CCFFFF;
+}
+
+#user-name {
+  border: 2px solid blue;
+  background-color:white;
+  border-radius: 10px;
+  font-family: sans-serif;
```

Comment:
You shouldn't need this since this will be inherited from the body CSS in main.css

Reply:
I'm a bit confused as to why we have two separate css pages - you said I shouldn't create a map.css file, but why do we have a separate user-page style sheet? Since the html file is linked to both stylesheets, can't we just combine them?

Reply:
I just meant the CSS targetting the body tag. I suggested moving the body color from index.html into main.css in another comment. I'm just saying that you don't need CSS targetting body here as well since we are already going to get what is in main.css

Reply:
Ohh, I see. So the body tag shouldn't be in the specific css files but in main? And it's okay that I have a map css file? (I deleted it thinking you wanted all the css in main but I can easily get it back).